tensorflow_lite_support/python/task/vision/image_classifier_test.py

```diff
@@ -0,0 +1,535 @@
+              }
+              classes {
+                index: 924
+                score: 0.00390625
+                class_name: "plate"
+              }
+              head_index: 0
+            }
+            """)
+        )
+      # Testing the model on burger_crop.jpg
+      self.assertEqual(
+        str(crop_result),
+        textwrap.dedent(
+          """\
+          classifications {
+            classes {
+              index: 934
+              score: 0.96484375
+              class_name: "cheeseburger"
+            }
+            classes {
+              index: 935
+              score: 0.0078125
+              class_name: "hotdog"
+            }
+            classes {
+              index: 119
+              score: 0.0078125
+              class_name: "Dungeness crab"
+            }
+            head_index: 0
+          }
+          """)
+      )
+    elif model_name == _MODEL_AUTOML:
+      if with_bounding_box:
+        # Testing the model on burger.jpg (w/ bounding box)
+        self.assertEqual(
+          str(image_result),
+          textwrap.dedent(
+            """\
+            classifications {
+              classes {
+                index: 2
+                score: 0.953125
+                class_name: "roses"
+              }
+              classes {
+                index: 0
+                score: 0.01171875
+                class_name: "daisy"
+              }
+              classes {
+                index: 1
+                score: 0.01171875
+                class_name: "dandelion"
+              }
+              head_index: 0
+            }
+            """)
+        )
+      else:
+        # Testing the model on burger.jpg (w/o bounding box)
+        self.assertEqual(
+          str(image_result),
+          textwrap.dedent(
+            """\
+            classifications {
+              classes {
+                index: 2
+                score: 0.96484375
+                class_name: "roses"
+              }
+              classes {
+                index: 4
+                score: 0.01171875
+                class_name: "tulips"
+              }
+              classes {
+                index: 0
+                score: 0.0078125
+                class_name: "daisy"
+              }
+              head_index: 0
+            }
+            """)
+        )
+      # Testing the model on burger_crop.jpg
+      self.assertEqual(
+        str(crop_result),
+        textwrap.dedent(
+          """\
+          classifications {
+            classes {
+              index: 2
+              score: 0.953125
+              class_name: "roses"
+            }
+            classes {
+              index: 0
+              score: 0.01171875
+              class_name: "daisy"
+            }
+            classes {
+              index: 1
+              score: 0.01171875
+              class_name: "dandelion"
+            }
+            head_index: 0
+          }
+          """)
+      )
+
+
+  @parameterized.parameters(
```

Comment:
If there's one test case, don't use parameterized test and just write the test code normally.